docs/tutorials/pachyderm-cat-dog.rst

```diff
@@ -9,9 +9,8 @@
 .. meta::
    :description: Follow along with this batch inferencing tutorial to see how to leverage Determined and Pachyderm together to streamline complex tasks.
 
-In this guide, we'll help you create a simple batch inferencing project in `Pachyderm
-<https://docs.pachyderm.com/latest/learn/glossary/pipeline/>`__, train your model using a Determined
```

Comment:
This link seems to work for me. 
See:

(tutorials/pachyderm-cat-dog: line   12) redirect  https://docs.pachyderm.com/latest/learn/glossary/pipeline/ - permanently to https://docs.pachyderm.com/products/mldm/latest/learn/glossary/pipeline/

Reply:
this one should not have been there